everest-testing/src/everest/testing/core_utils/controller/everest_test_controller.py

```diff
@@ -86,6 +86,26 @@ def diode_fail(self, connector_id=1):
             f"{self._mqtt_external_prefix}everest_external/nodered/{connector_id}/carsim/cmd/execute_charging_session",
             "sleep 1;iec_wait_pwr_ready;sleep 1;draw_power_regulated 32,3;sleep 5;diode_fail;sleep 36000;unplug")
 
+    def raise_error(self, connector_id=1):
```

Comment:
It would be nice if one could specify an error string here, it can still default to `MREC6UnderVoltage`